tests/driver.sh

```diff
@@ -259,8 +259,14 @@ int main()
 EOF
 
 # conditional operator
-# expr 10 "1 ? 10 : 5"
```

Comment:
Why didn't you re-use the existing test items?

Reply:
Because it is not complicated enough and it only cover "conditional operator in return statement". Current test bench covers three kind of usages.

Reply:
For profiling purpose, it would be better to have minimized test items available. Thus, it can reduce uncertainty.

Reply:
Test bench was reverted!

Reply:
You can append the new ternary operation test case as well.